frontend/src/features/public-form/PublicFormService.ts

```diff
@@ -155,46 +154,9 @@ export const submitEmailModeForm = async ({
   ).then(({ data }) => data)
 }
 
-export const submitStorageModeForm = async ({
-  formFields,
-  formLogics,
-  formInputs,
-  formId,
-  publicKey,
-  captchaResponse = null,
-  captchaType = '',
-  paymentReceiptEmail,
-  responseMetadata,
-  paymentProducts,
-  payments,
-}: SubmitStorageFormArgs) => {
-  const filteredInputs = filterHiddenInputs({
-    formFields,
-    formInputs,
-    formLogics,
-  })
-  const submissionContent = await createEncryptedSubmissionData({
-    formFields,
-    formInputs: filteredInputs,
-    publicKey,
-    responseMetadata,
-    paymentReceiptEmail,
-    payments,
-    paymentProducts,
-  })
-  return ApiService.post<SubmissionResponseDto>(
-    `${PUBLIC_FORMS_ENDPOINT}/${formId}/submissions/encrypt`,
-    submissionContent,
-    {
-      params: {
-        captchaResponse: String(captchaResponse),
-        captchaType,
-      },
-    },
-  ).then(({ data }) => data)
-}
-
-export const submitStorageModeClearForm = async ({
+// TODO (#5826): Fallback mutation using Fetch. Remove once network error is resolved
```

Comment:
can remove unused import `ENCRYPTION_BOUNDARY_SHIFT_SUBMISSION_VERSION` in line 5

Reply:
[24d15d6](https://github.com/opengovsg/FormSG/pull/6869/commits/24d15d6ba0d5211c473116da102367e78b2e9787)